setup.cfg

```diff
@@ -28,7 +28,7 @@ install_requires =
     colcon-core>=0.5.6
 packages = find:
 tests_require =
-    flake8
+    flake8>=3.6.0
```

Comment:
I'm guessing you're not also adding `<7` [like in `colcon-core`](https://github.com/colcon/colcon-core/blob/f7dfec7efc54b8d9a4ce0ba6e43c7137b5810011/setup.cfg#L47) (https://github.com/colcon/colcon-core/pull/608) because it's not necessary for this package?

Reply:
Indeed it seems to work fine with `7.0.0`, but [colcon/ci](https://github.com/colcon/ci) is still pinning it back until I've tested all the other extensions.